src/screens/TopicPageScreen/index.js

```diff
@@ -107,6 +106,12 @@ const TopicPageScreen = (props) => {
     extrapolate: 'clamp'
   });
 
+  const opacityImage = scrollY.interpolate({
+    inputRange: [0, 100],
+    outputRange: [0, 1],
+    extrapolate: 'clamp'
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new constant `opacityImage` has been introduced. It seems to be an animation value based on the scroll position. This change appears to be part of the visual enhancements mentioned in the PR summary. As long as `scrollY` is defined and is an instance of `Animated.Value`, this should work fine.



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
yap that's fine